sled-agent/src/illumos/running_zone.rs

```diff
@@ -301,6 +301,7 @@ impl RunningZone {
                 // Re-initialize guest_vnic state by inspecting the zone.
                 opte_ports: vec![],
                 link: None,
+                bootstrap_vnic: None, // TODO: Change this?
```

Comment:
I recommend looking at `get_control_interface` in `sled-agent/src/illumos/zone.rs` for how we're re-locating the `control_vnic` in this case. Basically, we ask the running zone for VNICs, and filter by prefix..

Reply:
Thanks! Fixed in https://github.com/oxidecomputer/omicron/pull/2320/commits/64b600f45337d7c58d54a5c5a25411aa60658dde